rct229/ruletest_engine/ruletest_jsons/section6/rule_6_1.json

```diff
@@ -0,0 +1,525 @@
+{
+    "rule-6-1-a": {
+        "Section": 6,
+        "Rule": 1,
+        "Test": "a",
+        "test_description": "Project has one building segment with two spaces. Both lighting power per area and lighting power allowance have been specified. The proposed building lighting power per area equals the maximum value of the proposed building's lighting power allowance and user building's lighting power per area.",
```

Comment:
Hey Yun Joon,

I think the test description is the same for all 6_1 rule jsons. Is it possible to update them to reflect the differences between the tests? That would help me see the intended differences!

Reply:
Thanks for finding this Juan. I just updated the test description parts.